liquibase-standard/src/test/groovy/liquibase/snapshot/jvm/ViewSnapshotGeneratorTest.groovy

```diff
@@ -0,0 +1,29 @@
+package liquibase.snapshot.jvm
+
+import liquibase.database.Database
+import liquibase.snapshot.DatabaseSnapshot
+import liquibase.snapshot.SnapshotControl
+import liquibase.structure.core.Schema
+import liquibase.structure.core.View
+import spock.lang.Specification
+
+class ViewSnapshotGeneratorTest extends Specification {
+
+    def "does not add any views to snapshot if database does not support views"() {
```

Comment:
probably need to update this or add a new test with view included but unsupported by db